docs/source/data/data_catalog.md

```diff
@@ -1,31 +1,34 @@
-# The Data Catalog
+# Introduction to the Data Catalog
 
-This section introduces `catalog.yml`, the project-shareable Data Catalog. The file is located in `conf/base` and is a registry of all data sources available for use by a project; it manages loading and saving of data.
+## The basics of `catalog.yml`
```

Comment:
It's always weird to me how h1 is smaller than h2 🤔 

![image](https://github.com/kedro-org/kedro/assets/316517/086261a0-c415-4dfc-9742-639b949809d2)